### ISSUE TITLE
Blank ads

### ISSUE THREAD

**Issue by ghost** [NodeBB-Community/nodebb-plugin-adsense#12]
Hi there,

I am experiencing an issue.  I got my adsense account,etc... but it is showing blank ads.

What am I doing wrong?

Thanks and kind regards,

**Comment by carlosjs23** [comment]
Same issue too.

**Comment by varundeva** [comment]
Same issue
Ads load only when page gets refresh

**Comment by aworobel** [comment]
Hi, 
Same issue, blank ads everywhere except the head one but as varundeva, does not refresh when consulting sections on the forum.

**Comment by fais3000** [comment]
@ghost @carlosjs23 @varundeva Have you trying disabling ajax based page loads? 

```
    $(document.body).off('click', 'a');
```

I found this to solve ad load issues for me.

**Comment by julianlam** [comment]
Yes, this will resolve the issue, in the sense that the site will no longer be a single-page app :smile: 

It's an effective workaround, but has its own issues.
